release/tune_tests/cloud_tests/workloads/run_cloud_test.py

```diff
@@ -1152,13 +1157,16 @@ def after_experiments():
 
     run_time = int(os.getenv("TUNE_RUN_TIME", "180")) or 180
 
+    run_start_timeout = 600 if "rllib" in os.environ["TUNE_TRAINABLE"] else 30
```

Comment:
cc @sven1977 @gjoliver 

the `_try_import_jax` takes forever. Previously the timeout was 30 seconds, but now we have to wait ~5 minutes until the trial actually starts.